platform/ios/PolyPodApp/PolyPodUITests/OnboardingTest.swift

```diff
@@ -0,0 +1,52 @@
+import XCTest
+
+class OnboardingTest: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+    }
+    
+    func testOnboardingShowsUpOnFirstRun() {
+        launchApp(firstRun: true)
+        assertOnboardingIsShown()
+    }
+    
+    func testOnboardingNotShownOnSubsequentRun() {
+        launchApp()
+        assertOnboardingIsNotShown()
+    }
+    
+    private func launchApp(firstRun: Bool = false, resetDefaults: Bool = true) {
```

Comment:
So if you're (maybe) using this elsewhere, did you consider putting it in its own helper module? Or maybe, since it's using `app`, its own class?

Reply:
Done!